src/Handlers/JsonServiceHandler.php

```diff
@@ -22,6 +22,8 @@
 
 class JsonServiceHandler extends JsonPostHandler
 {
+    public const DIRECT_RESPONSE = 'direct_response';
```

Comment:
Tags are global so it would be a good idea to introduce a namespace prefix. E.g. "allegro:jsonpost:direct_response".

Also, as this is JsonPostHandler's functionality, we should move the constant there. Otherwise it puts an unnecessary dependency from JsonPostHandler onto JsonServiceHandler.